src/node.cc

```diff
@@ -2581,9 +2569,26 @@ static void Binding(const FunctionCallbackInfo<Value>& args) {
     DefineConstants(env->isolate(), exports);
     cache->Set(module, exports);
   } else if (!strcmp(*module_v, "natives")) {
+    auto caller_script_id = GetCallerScriptId(env->isolate());
+    if (!IsInternalScriptId(env, caller_script_id)) {
+      // graceful-fs < 4 evals process.binding('natives').fs, which prohibits
+      // using internal modules in that module.  Encourage people to upgrade.
+      auto pid = getpid();
+      fprintf(stderr,
+              "(node:%d) process.binding('natives') is deprecated.\n", pid);
+      fprintf(stderr,
+              "(node:%d) If you use graceful-fs < 4, please update.\n", pid);
```

Comment:
@bnoordhuis yes, but see the list at https://github.com/nodejs/node/pull/2741#issuecomment-242514854.